lib/firebase.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:firebase/firebase.dart' as fb;
+import 'package:firebase/firestore.dart' as fs;
+
+class DB {
+  fs.DocumentReference _filtersRef;
+
+  DB() {
+    fb.initializeApp(
+        apiKey: "AIzaSyB3nbxl4cH5opeqmXSyRZMANoUMIlTch80",
+        authDomain: "avf-metrics.firebaseapp.com",
+        databaseURL: "https://avf-metrics.firebaseio.com",
+        projectId: "avf-metrics",
+        storageBucket: "avf-metrics.appspot.com",
+        messagingSenderId: "542922892166",
+        appId: "1:542922892166:web:cd7ae2802395a9f652c207");
+
+    fs.Firestore store = fb.firestore();
+    _filtersRef = store.collection("unit-chart").doc("filters");
+  }
+
+  Future<Map<String, dynamic>> readFilters() async {
```

Comment:
Does it make sense to make sense to call the model ctors here to return a set of objects rather than something closer to a JSON map?

Reply:
Ah yes, makes sense. Fixed!